lib/livebook/runtime.ex

```diff
@@ -195,7 +195,16 @@ defprotocol Livebook.Runtime do
   @type smart_cell_definition :: %{
           kind: String.t(),
           name: String.t(),
-          requirement: nil | %{name: String.t(), dependencies: list(dependency())}
+          requirement: nil | smart_cell_requirement()
```

Comment:
Which smart cell does not have requirements? 🤔

Reply:
All predefined cells have it and all cells discovered via `Kino.Smart.definitions()` don't :)

Reply:
Prrrrfect. Ship it.